test/solc/CommandLineInterface.cpp

```diff
@@ -1412,6 +1412,130 @@ BOOST_AUTO_TEST_CASE(cli_include_paths_ambiguous_import)
 	BOOST_REQUIRE(!result.success);
 }
 
+BOOST_AUTO_TEST_CASE(cli_ethdebug_no_ethdebug_in_help)
+{
+	OptionsReaderAndMessages result = runCLI({"solc", "--help"});
+	BOOST_REQUIRE(result.stdoutContent.find("ethdebug") == std::string::npos);
+}
+
+BOOST_AUTO_TEST_CASE(cli_ethdebug_incompatible_outputs)
+{
+	TemporaryDirectory tempDir(TEST_CASE_NAME);
+	createFilesWithParentDirs({tempDir.path() / "input.sol"});
+	// output selection "asm-json" not working with debug-info "ethdebug" or "ethdebug" output selection.
+	OptionsReaderAndMessages result = runCLI({"solc", "--debug-info", "ethdebug", "--asm-json", tempDir.path().string() + "/input.sol"}, "");
```

Comment:
```suggestion
	OptionsReaderAndMessages result;
```

Got a duplicate of this case on line 1430.

Reply:
You know, if Kamil was here, he'd say 'you should put this options into a `std::vector` and iterate through them in order to make the test more readable'. :D 

I will say though - it would make sense here, since we typically also assert the correctness of the error message (which you're not doing here).